sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md

```diff
@@ -3013,7 +3014,23 @@ Accept wildcard characters: False
 When set to `True`, forces a sync of **Content Security Policy** entries for SharePoint framework component in the tenant application catalog.
```

Comment:
this should say "for SharePoint Framework components". also, you are still using entries here. Wanna change it to sources too?